src/components/MeetYourHostLeft/MeetYourHostLeft.jsx

```diff
@@ -1,19 +1,23 @@
-import React from "react";
-import styles from "./PersonProfile.module.css";
+import styles from "./MeetYourHostLeft.module.css";
+import { BusinessIcon, LanguageIcon } from "../../icons";
 
-const PersonProfile = ({
-  title,
+const MeetYourHostLeft = ({
```

Comment:
You changed the name of this file from PersonProfile to MeetYourHostLeft, but PersonProfile is imported in App.jsx. It is better to change the name in App.jsx or completely remove it from there, as it doesn't need to be imported there.